alphalens/tears.py

```diff
@@ -191,139 +203,150 @@ def create_returns_tear_sheet(factor_data,
-                                         group_neutral)
+    factor_returns = perf.factor_returns(
+        factor_data, long_short, group_neutral
+    )
 
-    mean_quant_ret, std_quantile = \
-        perf.mean_return_by_quantile(factor_data,
-                                     by_group=False,
-                                     demeaned=long_short,
-                                     group_adjust=group_neutral)
+    mean_quant_ret, std_quantile = perf.mean_return_by_quantile(
+        factor_data,
+        by_group=False,
+        demeaned=long_short,
+        group_adjust=group_neutral,
+    )
 
-    mean_quant_rateret = \
-        mean_quant_ret.apply(utils.rate_of_return, axis=0,
-                             base_period=mean_quant_ret.columns[0])
+    mean_quant_rateret = mean_quant_ret.apply(
+        utils.rate_of_return, axis=0, base_period=mean_quant_ret.columns[0]
+    )
 
-    mean_quant_ret_bydate, std_quant_daily = \
-        perf.mean_return_by_quantile(factor_data,
-                                     by_date=True,
-                                     by_group=False,
-                                     demeaned=long_short,
-                                     group_adjust=group_neutral)
+    mean_quant_ret_bydate, std_quant_daily = perf.mean_return_by_quantile(
+        factor_data,
+        by_date=True,
+        by_group=False,
+        demeaned=long_short,
+        group_adjust=group_neutral,
+    )
 
     mean_quant_rateret_bydate = mean_quant_ret_bydate.apply(
-        utils.rate_of_return, axis=0,
-        base_period=mean_quant_ret_bydate.columns[0]
+        utils.rate_of_return,
+        axis=0,
+        base_period=mean_quant_ret_bydate.columns[0],
     )
 
-    compstd_quant_daily = \
-        std_quant_daily.apply(utils.std_conversion, axis=0,
-                              base_period=std_quant_daily.columns[0])
+    compstd_quant_daily = std_quant_daily.apply(
+        utils.std_conversion, axis=0, base_period=std_quant_daily.columns[0]
+    )
 
-    alpha_beta = perf.factor_alpha_beta(factor_data,
-                                        factor_returns,
-                                        long_short,
-                                        group_neutral)
+    alpha_beta = perf.factor_alpha_beta(
+        factor_data, factor_returns, long_short, group_neutral
+    )
 
-    mean_ret_spread_quant, std_spread_quant = \
-        perf.compute_mean_returns_spread(mean_quant_rateret_bydate,
-                                         factor_data['factor_quantile'].max(),
-                                         factor_data['factor_quantile'].min(),
-                                         std_err=compstd_quant_daily)
+    mean_ret_spread_quant, std_spread_quant = perf.compute_mean_returns_spread(
+        mean_quant_rateret_bydate,
+        factor_data["factor_quantile"].max(),
+        factor_data["factor_quantile"].min(),
+        std_err=compstd_quant_daily,
+    )
```

Comment:
Mostly stylistic. I made a few changes to the plotting of cumulative returns since cumulative returns are now only being computed for one frequency: https://github.com/quantopian/alphalens/pull/361/files#diff-5d20acb213bbb2b2824b8cf64b09e0beR279